Doc/reference/datamodel.rst

```diff
@@ -1008,6 +1008,7 @@ Special attributes:
 
    :attr:`__firstlineno__`
       The line number of the first line of the class definition, including decorators.
+      Setting :attr:`__module__` removes :attr:`__firstlineno__`.
```

Comment:
```suggestion
      Setting :attr:`!__module__` removes :attr:`!__firstlineno__`.
```

Note that the link to `__module__` does not exist (we don't have a global `.. attribute:: __module__`). We can address this issue in a follow-up PR, but for now, let's just suppress the link.